src/theme/bootstrap-override/_bootstrap-italia-site.scss

```diff
@@ -205,4 +205,5 @@
   @import 'bootstrap-italia/src/scss/utilities/focus';
   @import 'bootstrap-italia/src/scss/utilities/icons';
   @import './bootstrap-italia/focus';
+  @import './bootstrap-italia/avatar';
```

Comment:
Questo meglio se lo metti subito dopo all'import dell'avatar originale (riga 142?), poi non ho capito se l'hai rimpiazzato del tutto, allora in caso togli l'import originale, oppure se hai solo aggiunto alcuni stili per personalizzare l'avatar originale, in tal caso va bene lasciare anche l'import originale

Reply:
L'idea era di tenere negli override la modifica agli stili solo per `.avatar.size-lg` senza rimpiazzare. Se non dovevo fare cosi', ma riportare tutto il file e cambiare solo le cose che mi servivano invece, allora modifico l'override. 
Non mi ricordo benissimo i criteri di come si agiva qui per gli override di bootstrap-italia